backend/routes/meta.js

```diff
@@ -273,14 +361,43 @@
 			});
 	}
 
+	function queryMetadataByRules(req, res, next) {
+		const rules = req.body;
+		let branch;
+
+		if (!req.params.rev) {
+			branch = C.MASTER_BRANCH_NAME;
+		}
+
+		const showMeshIds = (req.query.meshids) ? JSON.parse(req.query.meshids) : false;
+
+		if (showMeshIds) {
+			Meta.getMeshIdsByRules(req.params.account, req.params.model, branch, req.params.rev, rules)
+				.then(obj => {
+					responseCodes.respond(utils.APIInfo(req), req, res, next, responseCodes.OK, obj, undefined, undefined);
```

Comment:
you don't need to pass undefineds if they're at the end of the function calls. by the virtue of not passing over something they'd be undefined.
```suggestion
					responseCodes.respond(utils.APIInfo(req), req, res, next, responseCodes.OK, obj);
```